documentation/api/query/v4/index.markdown

```diff
@@ -30,6 +30,9 @@ operator or a [PQL][pql] query string.
 a query with a [`from`][from] is required to choose the [entity][entities] for which to query. For
 general info about queries, see [our guide to query structure.][query]
 
+* `parse_pql_only`: optional. A boolean value. When true, the query response 
```

Comment:
@wkalt If the query is in AST, it still returns AST. I can add that to the documentation.
As for what to call it, I know what you mean, but it's really only useful for parsing PQL into AST. Like, if someone were to use it with an AST query, nothing would break, but it's not what the param is meant to be used for.
Still think I should change it?